services/API-service/README.md

```diff
@@ -1,114 +1,16 @@
 # IBF-API-service: REST API of the IBF-system
 
-This service creates endpoints which serves data from the IBF-database to (for example) the IBF-interfaces.
+The [IBF API](https://ibf.510.global/docs) is the REST API of the IBF-system, which allows the IBF pipelines (a.o.) to write data to the database and the [IBF dashboard](https://ibf.510.global) (a.o.) to read data from the database.
 
----
+## Getting Started
 
-## Getting started / Installation
+- See [main README] (https://github.com/rodekruis/IBF-system/blob/master/README.md)
```

Comment:
@jannisvisser Please remove the breaking whitespace between [main README] and (url)